locust/test/test_parser.py

```diff
@@ -32,3 +33,16 @@ def test_skip_log_setup(self):
         ]
         opts = self.parser.parse_args(args)
         self.assertEqual(opts.skip_log_setup, True)
+
+    def test_parameter_parsing(self):
+        conf_file = '/tmp/locust.conf'
```

Comment:
Can we use NamedTemporaryFile here? (https://docs.python.org/2/library/tempfile.html#tempfile.NamedTemporaryFile)

Reply:
Sure, I'll change that!